backend/redis_client.py

```
@@ -0,0 +1,398 @@
+        Returns:
+            True if registered, False otherwise
+        """
+        if not self.connected:
+            return False
+
+        try:
+            key = f"astra:resilience:vote:{instance_id}"
+            vote["timestamp"] = datetime.utcnow().isoformat()
+            await self.redis.set(key, json.dumps(vote), ex=ttl)
+            logger.debug(f"Registered vote from {instance_id}")
+            return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)